EdvClient.js

```diff
@@ -637,11 +643,13 @@ export class EdvClient {
    * @param {string} options.config - The EDV's configuration.
    * @param {https.Agent} [options.httpsAgent=undefined] - An optional
    *   node.js `https.Agent` instance to use when making requests.
+   * @param {defaultHeaders} [options.defaultHeaders=undefined] - An optional
+   *   defaultHeaders object to use when making requests.
    *
    * @return {Promise<Object>} resolves to the configuration for the newly
    *   created EDV.
    */
-  static async createEdv({url = '/edvs', config, httpsAgent}) {
+  static async createEdv({url = '/edvs', config, httpsAgent, defaultHeaders}) {
```

Comment:
For all the static methods it looks like it should just be named "headers".

Reply:
+1 to @davidlehn

Reply:
agree, implemented